src/components/CalculationOptions/CalculationOptions.tsx

```diff
@@ -36,7 +40,11 @@ function CalculationOptionsButtons() {
       <FormControlLabel
         control={
           <Checkbox
-            checked={outputType !== 'rawResults' && calculationOptions.calculateHTML}
+            checked={
+              outputType !== 'rawResults' &&
+              calculationOptions.reportType !== 'summary' &&
+              calculationOptions.calculateHTML
+            }
```

Comment:
These conditionals are starting to get a little long to repeat. Maybe break this out into a shouldBeChecked or isChecked function?